### DISCUSSION TITLE
Lab 1 Questions

### DISCUSSION THREAD

**Discussion by jason-muljadi** [albertkun/21S-ASIAAM-191A#26] [Help Desk] [answered]
Hello all! Two questions that I have:

1. How do we open our GeoJSON file in order to check our work?
2. For the write-up, question 2 asks "Why you used these fields." Are we supposed to give a technical AND personal explanation for our selection?

**Comment by albertkun** [comment, marked as answer]
Hello @jason-muljadi good questions and here are some answers:

1. You have three options for checking the GeoJSON:
  - Add the GeoJSON in QGIS to see if it appears (it should behave just like a shapefile!). 
  - Open the file in VS Code and it will appear as human readable text. 
  - Preview the GeoJSON online in your GitHub Lab Assignments repository.

2. Mainly that was designed to ask about your personal reason for choosing the fields. A technical explanation wouldn't hurt, but isn't necessary.

Hope that helps!

P.S. Remember to mark the question as answered if it does!
